### PR TITLE
Add make bump-kubevirtci target

### PR DESCRIPTION
**Pull request by oshoval**
Signed-off-by: Or Shoval <oshoval@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -135,6 +135,9 @@ gen-k8s-check: $(apis_sources)
 components:
 	./hack/components.sh
 
+bump-kubevirtci:
+	./hack/bump-kubevirtci.sh
```
Comment (oshoval) [inline Makefile]: to rm -rf the folder also ?

Comment (phoracek) [reply Makefile]: Yes. Without it, it won't update.

File: Makefile [hunk 2]
```diff
@@ -135,6 +135,9 @@ gen-k8s-check: $(apis_sources)
...
+
 .PHONY: \
```
Comment (qinqon) [inline Makefile]: .PHONY is missing

File: Makefile (hunk 1 above)
Comment (oshoval) [reply Makefile]: thx

**Comment by oshoval** [comment]
failures are of course not due to this PR

**Comment by oshoval** [comment]
do we need to override or the kubevirt bot will retest and finally merge it once it see its flaky ?
same question for #304

**Comment by phoracek** [comment]
/override pull-e2e-cluster-network-addons-operator-workflow-k8s

**Comment by phoracek** [comment]
/override pull-e2e-cluster-network-addons-operator-workflow-k8s

**Comment by phoracek** [comment]
/override pull-e2e-cluster-network-addons-operator-workflow-k8s

**Pull request merged**